cli/src/wallet.rs

```diff
@@ -849,13 +849,11 @@ pub fn process_find_program_derived_address(
     seeds: &Vec<Vec<u8>>,
     program_id: &Pubkey,
 ) -> ProcessResult {
-    if config.verbose {
-        println!("Seeds: {seeds:?}");
-    }
     let seeds_slice = seeds.iter().map(|x| &x[..]).collect::<Vec<_>>();
     let (address, bump_seed) = Pubkey::find_program_address(&seeds_slice[..], program_id);
     let result = CliFindProgramDerivedAddress {
         address: address.to_string(),
+        seeds: seeds.clone(),
```

Comment:
clippy is complaining about the `&Vec` passed in, and saying to change that to `&[_]` instead: https://github.com/anza-xyz/agave/actions/runs/11114470308/job/31517664360?pr=3042#step:5:1336

And then this will need to change to `seeds.to_owned()`